generator/test_case_generator.go

```diff
@@ -44,8 +44,10 @@ var testTypeToTestDirMap = map[string][]string{
 	},
 	"ec2_performance": {
 		"../../test/performance/logs",
+		"../../test/performance/statsd",
```

Comment:
The test case generator will generate test case based on: [the combination of host resources information](https://github.com/aws/amazon-cloudwatch-agent-test/blob/main/generator/resources/ec2_stress_test_matrix.json#L1-L42) and the [test cases](https://github.com/aws/amazon-cloudwatch-agent-test/blob/main/generator/test_case_generator.go#L48-L50). Afterwards, the combination will be [used by the Github Workflow](https://github.com/aws/amazon-cloudwatch-agent/blob/main/.github/workflows/integrationTest.yml#L139)  to [generate the matrix for running the appropriate tests](https://github.com/aws/amazon-cloudwatch-agent/blob/main/.github/workflows/integrationTest.yml#L819-L820)